.github/workflows/test_s3.yml

```diff
@@ -17,11 +17,16 @@ jobs:
       MINIO_ROOT_PASSWORD: password
       MINIO_URL: http://localhost:9000
       MINIO_ALIAS: myminio
+      MINIO_BUCKET: acquire-test
       MINIO_ACCESS_KEY: acquire
       MINIO_SECRET_KEY: 12345678
 
     steps:
       - uses: actions/checkout@v3
+        with:
+          submodules: true
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Which submodule do we need to check out again for this PR?

Reply:
We need to check out acquire-common. We weren't building before this, so we didn't need it.